src/store/modules/user/index.ts

```diff
@@ -13,7 +13,12 @@ const userModule: Module<UserState, RootState> = {
       currentFacility: {},
       currentEComStore: {},
       instanceUrl: '',
-      facilityLocationsByFacilityId: {}
+      facilityLocationsByFacilityId: {},
+      productIdentificationPref: {
+        primaryId: 'productId',
+        secondaryId: '',
+        fromDate: ''
```

Comment:
I think we should not store from date in state, it should be taken care while updating the value.

Reply:
Removed storing fromDate in state and fetching fromDate whenever updating the id prefs.